### PR TITLE
Sch 211658 tandc

### PR DESCRIPTION
**Pull request by shewood**
## [ADO-211658](https://dev.azure.com/VP-BD/DECD/_workitems/edit/211658)

[AB#211658](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/211658)

### Changelog - "fix:" for bug fixes, "feat:" for features. Read more about Conventional Commits at https://www.conventionalcommits.org/en/v1.0.0/#summary
feat: use ecas links for t&c

### Description of proposed changes:
use ecas links for t and c and privacy in the footer. 
removed references to the page. 

### What to test for/How to test
test footer links to make sure they work (need auth for it)

### CODE REVIEW

File: locales/en.js [hunk 1]
```diff
@@ -125,12 +125,18 @@ export default {
   footerAboutURL: 'https://www.canada.ca/en/government/about.html',
   footerAbout: 'About Canada.ca',
 
+  footerTermsAndConditionURLtest:
```
Comment (krischarbonneau) [inline locales/en.js:128]: I'm not a fan of this naming. Could we instead call it something along the lines of `footerTermsAndConditionsECAS`? Same with the privacy URL.

File: locales/fr.js [hunk 2]
```diff
@@ -120,12 +120,17 @@ export default {
...
+  footerTermsAndConditionURLtest:
+    'https://srv241-s2.sade.hrdc-drhc.gc.ca/ecas-seca/rascl_ii/SCL/TC.aspx?mode=ReadOnly&lang=fra',
   footerTermsAndConditionURL:
-    '/fr/avis-confidentialite-modalites#termesEtConditions',
+    'https://srv136.services.gc.ca/ecas-seca/rascl/SCL/TC.aspx?mode=ReadOnly&lang=fra',
   footerTermsAndCondition: 'Avis',
   footerTermsAndConditionAnchor: 'termesEtConditions',
 
-  footerPrivacyURL: '/fr/avis-confidentialite-modalites/',
+  footerPrivacyURLtest:
```
Comment (krischarbonneau) [inline locales/fr.js:130]: Could we call this `footerPrivacyDevURL` instead, or something along those lines? I'm not a huge fan of this naming sense

**Review by krischarbonneau — COMMENTED** [review]
Looks fine for the most part, just a comment on the naming sense for the dev URLS

File: locales/fr.js (hunk 2 above)
Comment (shewood) [reply locales/fr.js:130]: Yes, of course. I will update it.

**Review by krischarbonneau — APPROVED** [review]
LGTM

**Pull request merged**